src/js/view/controls/info-overlay.js

```diff
@@ -6,7 +6,9 @@ import { cloneIcon } from 'view/controls/icons';
 import { timeFormat } from 'utils/parser';
 
 export default function (container, model, api, onVisibility) {
-    const template = createElement(InfoOverlayTemplate());
+    const analytics = api.getPlugin('jwpsrv');
+    const shouldTrackUser = analytics && !analytics.doNotTrackUser();
```

Comment:
Looks like there's an assumption here that `doNotTrackUser()` will _always_ exist in the ping plugin. I can make sure that's the case -- I imagine this adds negligible overhead so it should be OK. There's an alternate path, I think, where we only call `doNotTrackUser()` in cases where the player config has the `doNotTrack` option, and then only the "do not track" version of the plugin has the function exposed. But that may be needlessly complex, I think this is fine. 

LGTM!

Reply:
we should absolutely not assume it will always be present - in case we ever push plugins out over older versions or mistag (we have done this in the past)

Reply:
good point, thanks Joe. In that case let's make sure to check for the existence of the function before calling it!